addons/maintenance/migrations/12.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,41 @@
+---Fields in module 'maintenance'---
+maintenance  / maintenance.equipment    / activity_date_deadline (date) : now a function
+maintenance  / maintenance.equipment    / company_id (many2one)         : NEW relation: res.company
+maintenance  / maintenance.equipment    / effective_date (date)         : NEW required: required, req_default: function
```

Comment:
Fill this with `date(create_date)` for having the same behavior than previously.